packages/x-components/src/design-system/components/input-group/default.scss

```diff
@@ -74,15 +76,22 @@
     flex: 1 1 100%;
     min-width: 0;
     border: none;
```

Comment:
I dont know why, but now we are showing the outline

![image](https://user-images.githubusercontent.com/6597815/195363765-66d90e93-6af6-4a56-9bc7-daa9d2fef2b3.png)

Reply:
Yes, it was already "visible" but covered by the Clear button; now that this button is hidden when there is no query the outline can be easily perceived.